### PR TITLE
My enumerable

### PR DESCRIPTION
**Pull request by faizi2500**
### Following tasks were achieved in this project:

- [x] No linter errors
- [x] Proper documentation
- [x] Pair Programming
- [x] Create a class MyList that has an instance variable @list
- [x] implement a method #each that yields successive members of @list and uses the MyEnumerable module in the MyList
- [x] Create a module MyEnumerable that implements
- #all?
- #any?
- #filter?
-  [x] Each class and module should have a separate .rb file

### CODE REVIEW

File: list.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+require_relative 'enumerable'
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
```
Comment (rloterh) [inline list.rb]: - [ ] The _constructor_ of the `MyList` class is not able to accept multiple arguments. `MyList` class should be able to print output of a new initialization such as: `p MyList.new(4,6,8)`. Kindly ensure that the constructor of `MyList` class can arguments by implementing your constructor similar to this:
```ruby
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Faizii & Amine, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- Descriptive Pull Request
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: list.rb (hunk 1 above)
Comment (medaminedev66) [reply list.rb]: Thanks, @rloterh, for reviewing our pull request. Yes, it's a mistake. We thought that the input is an array. We will correct that and commit it again.

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**